### PR TITLE
Set default Gemfile

### PR DESCRIPTION
**Pull request by dkubb**
This branch sets the default Gemfile, upgrades dependencies, and cleans up issues preventing the example rails app from executing properly.

### CODE REVIEW

**Comment by dkubb** [comment]
@mbj this is ready for review.

File: alpine-rails-nginx/Dockerfile [hunk 1]
```diff
@@ -31,10 +31,10 @@ COPY nginx-remove-server-name.patch /tmp/
 RUN /tmp/install-nginx.sh \
   && rm -rf -- /usr/local/src /tmp/*
 
+# Setup bundler for the root user
+RUN bundle config --global -- gemfile '/opt/rails/Gemfile' \
+  && bundle config --global -- deployment
```
Comment (mbj) [inline alpine-rails-nginx/Dockerfile:36]: You probably meant `--deployment` not `-- deployment` (note the whitespace difference).

**Comment by mbj** [comment]
@dkubb I found a validation blocking flag https://github.com/dkubb/dockerfiles/pull/21#discussion_r46494084.

I'm fine with the rest of the diff.

Should we have a CI building docker images? Low hanging error fruits, and simple typos could be found faster than with a review.

File: alpine-rails-nginx/Dockerfile (hunk 1 above)
Comment (dkubb) [inline alpine-rails-nginx/Dockerfile:36]: No, the `bundle config` commands don't include the `--` prefix. The `--` is the separator between the command options and command arguments.

Comment (mbj) [inline alpine-rails-nginx/Dockerfile:36]: Ohh, I see. That makes sense. Sorry for the noise.

**Pull request merged**